api/src/wfl/service/postgres.clj

```diff
@@ -67,13 +57,17 @@
            (remove (comp finished? :status))
            (run! #(update! % (get-status! workload %)))))))
 
-(def update-workflow-statuses!
-  "Use `tx` to update `status` of Cromwell `workflows` in a `workload`."
-  (letfn [(get-cromwell-status [{:keys [cromwell]} {:keys [uuid]}]
-            (if (util/uuid-nil? uuid)
-              "skipped"
-              (cromwell-status cromwell uuid)))]
-    (make-update-workflows get-cromwell-status)))
+(defn update-workflow-statuses!
+  "Use `tx` to update `status` the workflows in a `workload`."
+  [tx {:keys [cromwell uuid items]}]
+  (let [uuid->status (->> {:label (str "workload:" uuid) :includeSubworkflows "false"}
+                          (cromwell/query (first (all/cromwell-environments cromwell)))
```

Comment:
Why was re-introducing environments necessary?

Reply:
The function I'm calling operates on environments, and the function is called from a lot of contexts where only the environment keyword is available. Changing `query` to not use environments is totally possible, just a larger refactoring outside of this change that would affect a lot of other code.